cmake/Modules/OpmSatellites.cmake

```diff
@@ -297,9 +297,9 @@ macro(opm_add_test TestName)
       if (CURTEST_DRIVER)
         set(CURTEST_COMMAND ${CURTEST_DRIVER} ${CURTEST_DRIVER_ARGS} ${CURTEST_EXE_NAME} ${CURTEST_TEST_ARGS})
       else()
-        set(CURTEST_COMMAND "${PROJECT_BINARY_DIR}/bin/${CURTEST_EXE_NAME}")
+        set(CURTEST_COMMAND ${PROJECT_BINARY_DIR}/bin/${CURTEST_EXE_NAME})
```

Comment:
> a list is `;` separated, not `" "` separated

Good point.  I guess there are still corner cases, but I don't expect anyone to
```sh
mkdir -p "c:;;;;/just/'cause/I can"
```
on their own build machines.